### PR TITLE
[WEB3-52] Migrate inline and ts styles to sass

### PR DESCRIPTION
**Pull request by ti-martins**
---
name: "Migrate inline and ts styles to sass"
about: "Use this template for requesting merging of new code changes"
title: "[WEB3-52] Migrate inline and ts styles to sass"
---

# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-52

## Type of change

- Enhancement

## How Has This Been Tested?

Comparing each component's styles to previous state

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/react-app/src/styles/edit.scss [hunk 1]
```diff
@@ -65,6 +65,21 @@
   color: #cc6161;
 }
 
+.image-input-filled {
+  width: 100%;
+  max-height: 250px;
+  cursor: pointer;
```
Comment (goncaloadolfo) [inline packages/react-app/src/styles/edit.scss:71]: Maybe I am missing something, but is the image clickable?

File: packages/react-app/src/components/giveaways/details/MainContent.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ const GiveawayMainContent = ({ participants }: GiveawayMainContentProps) => {
   return (
     <Stack sx={{ paddingLeft: { sm: '80px', lg: '0px' } }}>
       <Typography
+        className="giveaway-title"
         sx={{
           fontSize: '35px',
           fontWeight: 'bold',
```
Comment (goncaloadolfo) [inline packages/react-app/src/components/giveaways/details/MainContent.tsx:46]: I believe this one was missed

File: packages/react-app/src/styles/edit.scss (hunk 1 above)
Comment (ti-martins) [reply packages/react-app/src/styles/edit.scss:71]: Yes, it renders 2 different components based on if theres as image uploaded or not, but it is always clickable 👍

File: packages/react-app/src/components/giveaways/details/MainContent.tsx (hunk 2 above)
Comment (ti-martins) [reply packages/react-app/src/components/giveaways/details/MainContent.tsx:46]: Most of the sx props were intentionally not changed due to sx and style props working differently and the replacement not being as clean as just passing it className. In this case one possible solution would be to find out what css selectors we need and adjust styling accordingly, however that may generate a bigger code footprint unnecessarily. I'll double check this case either way 👍

**Review by goncaloadolfo — APPROVED** [review]

**Review by marciammart — APPROVED** [review]

**Pull request merged**